src/strategies/erc20-balance-of-weighted.ts

```diff
@@ -0,0 +1,19 @@
+import erc20BalanceOf from './erc20-balance-of';
+import { StrategyParams } from './index';
+
+export default async function getValue(
+  params: StrategyParams,
+  network: number,
+  snapshot: number
+): Promise<number> {
+  return (
+    (await erc20BalanceOf(
+      {
+        ...params,
+        address: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984'
+      },
+      network,
+      snapshot
+    )) * params.weight
```

Comment:
Shouldn't this be divided by the weight instead of multiplied?

Reply:
https://github.com/snapshot-labs/snapshot-strategies/blob/91abd258f10b4e881375890647bc0b5c58bd1cfb/src/strategies/erc20-balance-of-weighted/index.ts#L23 

Seems like a multiplication